From f2a9c9fdef0aa0aa55d42768a883e61f7134aedc Mon Sep 17 00:00:00 2001
From: ywan171 <yi.a.wang@intel.com>
Date: Fri, 20 Jul 2012 18:37:47 +0800
Subject: [PORT FROM MAIN] native: add intel color format

BZ: 47307

add intel color format
Signed-off-by: ywan171 <yi.a.wang@intel.com>

Change-Id: I4fad7b3aaa17c84c2166e6119940963bf3d1c4c3
Orig-Change-Id: Id3e455e7c386497fccb5d320ce179a380eeb7540
Reviewed-on: http://android.intel.com:8080/74483
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 include/media/openmax/OMX_IVCommon.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 include/media/openmax/OMX_IVCommon.h

diff --git a/include/media/openmax/OMX_IVCommon.h b/include/media/openmax/OMX_IVCommon.h
old mode 100644
new mode 100755
index 85bf00d..dd3ed9c
--- a/include/media/openmax/OMX_IVCommon.h
+++ b/include/media/openmax/OMX_IVCommon.h
@@ -158,6 +158,7 @@ typedef enum OMX_COLOR_FORMATTYPE {
      * */
     OMX_COLOR_FormatAndroidOpaque = 0x7F000789,
     OMX_TI_COLOR_FormatYUV420PackedSemiPlanar = 0x7F000100,
+    OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar =  0x7FA00E00,
     OMX_QCOM_COLOR_FormatYVU420SemiPlanar = 0x7FA30C00,
     OMX_QCOM_COLOR_FormatYUV420PackedSemiPlanar64x32Tile2m8ka = 0x7FA30C03,
     OMX_SEC_COLOR_FormatNV12Tiled = 0x7FC00002,
-- 
1.7.4.1

